### DISCUSSION TITLE
Question about %townyadvanced_nation_tag%

### DISCUSSION THREAD

**Discussion by JonatanLaMerce** [TownyAdvanced/Towny#5284] [Q&A] [answered]
Hi, I have a question about the %townyadvanced_nation_tag% placeholder.

My nation is called ASDF:
When I run a /ta nation ASDF rename ASD or /nation set name ASD

The placeholder %townyadvanced_nation_tag% of the chat/tab/tag is still in ASDF instead of updating to ASD.

even restarting the server or logging out the user and logging in again still the same TAG of previous nation, only if it is changed for example within a menu with the deluxeMenus. 

Is it a bug or should I configure or do something to display the changed tag?

**Comment by LlmDl** [comment, marked as answer]
/n set name NAME isn't what is shown in the tag placeholder, which is set using /n set tag TAG.
